roles/attach-ib/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+ib_mods:
```

Comment:
Are these no longer required now that we're relying on the RDMA service to load them?

Reply:
They have to be on the system to load them. I think your modifications to the Atomic image should handle the installation of them, and you can just rely on the rdma service being enabled to load them.